limitador/build.rs

```diff
@@ -0,0 +1,20 @@
+use std::error::Error;
+use std::path::Path;
+
+fn main() -> Result<(), Box<dyn Error>> {
+    generate_protobuf()
```

Comment:
This probably should be conditional on the feature being enabled... see `cfg!`, such as [here](https://github.com/Kuadrant/limitador/blob/main/limitador-server/build.rs#L37-L39)

Reply:
will do.